Make resource buttons work when instance is running

Scrumplex opened the pull request:
We enabled the buttons for most ExternalResource pages before, but seems like someone forgor removing the code that also prevented the actions from actually doing stuff.

Scrumplex committed:
fix: make resource buttons work when instance is running

Signed-off-by: Sefa Eyeoglu <contact@scrumplex.net>

flowln reviewed (commented):
The models still have the logic to disable interactions when the instance is running, so the buttons are now not grayed out, but don't do anything when the instance is running :thinking:

Scrumplex commented:
> The models still have the logic to disable interactions when the instance is running, so the buttons are now not grayed out, but don't do anything when the instance is running thinking

Uh, that was exactly the thing I was trying to fix here.

See #59 which only prevented the buttons from being disabled, but didn't actually allow the functions to do anything.

flowln committed:
fix: don't disable interaction on non-mod models when inst. is running

It is still useful for mods though, since we don't wnt to allow
disabling mods when the instance is running, as that wouldn't affect the
running instance, possibly causing confusion

Signed-off-by: flow <flowlnlnln@gmail.com>

flowln reviewed (approved):
boop :)

Scrumplex commented:
Oh. Why did we have that code before. Sounds like a bad idea to have it in MinecraftInstance :eyes:

flowln commented:
well, i guess it is connecting to the instance's signal after all, i don't think there's a much better place for that. i may be wrong though :thinking:

Scrumplex commented:
> well, i guess it is connecting to the instance's signal after all, i don't think there's a much better place for that. i may be wrong though thinking

Oh, yeah never mind. I thought we were touching the ExternalResourcePage here, but if it was touching the model, it was fine :D
